README.md

```diff
@@ -27,11 +27,34 @@ cd notebooks
 git pull
 ``` 
 
+## Notebooks Table of Contents
+
+This repository contains a collection Jupyter notebooks that teach you how to use Planet Insights Platform. 
+
+You can find a [table of contents here](/jupyter-notebooks/).
+
 ## Authentication
 
-## Access Your Planet API Key in Python
+To use these notebooks, which interact with [Planet's APIs](https://docs.planet.com/develop/apis), you will need to have an account to authenticate. You can find our  [authentication documentation here](https://docs.planet.com/develop/authentication/).
+
+For the quickest way to get started, you can log in with the CLI using the follow command:
+
+```bash
+planet auth login
+```
+
+Or from a Python environment with access to a web browser, you can run the following:
+
+```python
+import planet
+auth = planet.Auth.from_profile('planet-user', save_state_to_storage=True)
+if not auth.is_initialized():
+    auth.user_login(allow_open_browser=True, allow_tty_prompt=True)
+```
```

Comment:
I like the intro, don't think we need the full login block here (lines 40 - 53), for a couple reasons:
 - Collab and docker notebooks operate in an isolated environment, so logging in won't affect them
 - Any notebooks that actually have you login already explain this